token/program-2022/src/pod.rs

```diff
@@ -0,0 +1,99 @@
+//! Rewrites of the base state types represented as Pods
+
+use {
+    crate::{instruction::MAX_SIGNERS, state::PackedSizeOf},
+    bytemuck::{Pod, Zeroable},
+    solana_program::{program_pack::IsInitialized, pubkey::Pubkey},
+    spl_pod::primitives::PodU64,
+};
+
+/// Mint data stored as a Pod type
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Pod, Zeroable)]
+pub struct PodMint {
+    /// Optional authority used to mint new tokens. The mint authority may only
+    /// be provided during mint creation. If no mint authority is present
+    /// then the mint has a fixed supply and no further tokens may be
+    /// minted.
+    pub mint_authority: PodCOption<Pubkey>,
+    /// Total supply of tokens.
+    pub supply: PodU64,
+    /// Number of base 10 digits to the right of the decimal place.
+    pub decimals: u8,
+    /// Is not 0 if this structure has been initialized
```

Comment:
Just "not 0"? Or "exactly 1"?
Also, don't we have a PodBool in spl-pod? Any reason not to use that?

Reply:
Yeah sure, can do!